src/main/java/com/google/api/generator/gapic/composer/ServiceClientClassComposer.java

```diff
@@ -108,8 +109,10 @@ public static ServiceClientClassComposer instance() {
     return INSTANCE;
   }
 
-  @Override
-  public GapicClass generate(Service service, Map<String, Message> messageTypes) {
+  public GapicClass generate(
+      Service service, Map<String, Message> messageTypes, Map<String, ResourceName> resourceNames) {
+    // TODO(miraleung): Clean up the hierarchy to avoid pass another parameter (resourceNames is
+    // only used for composing sample code). See b/174257081.
```

Comment:
As noted in #512, could we please keep this public codebase clean of internal bug numbers? While we're here, could we please update the bug's content as well?

Reply:
sg